ql/src/java/org/apache/hadoop/hive/ql/metadata/HiveUtils.java

```diff
@@ -302,6 +317,52 @@ public static String unparseIdentifier(String identifier) {
     return unparseIdentifier(identifier, Quotation.BACKTICKS);
   }
 
+  public static UnparseTranslator collectUnescapeIdentifierTranslations(ASTNode node)
```

Comment:
no change here, just the class move, right?

Reply:
yes